.storybook/main.ts

```diff
@@ -33,6 +33,14 @@ module.exports = {
       '@onfido/castor-react': resolve(__dirname, '../packages/react/src'),
     };
 
+    // instrument source code if running e2e tests
```

Comment:
[Minor] If you agree with earlier, then:

```suggestion
    // instrument source code if running E2E tests
```